.github/workflows/ci_cron_weekly.yml

```diff
@@ -145,7 +146,10 @@ jobs:
                                   liberfa1
 
           run: |
+            uname -a
+            echo "LONG_BIT="$(getconf LONG_BIT)
```

Comment:
@pllim This should return
```
LONG_BIT=32
```
or
```
LONG_BIT=64
```
Does that seem easier to pick out in the log?